assembl/static2/js/app/store/configureStore.dev.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+
+export default function configureStore(initialState, rootReducer, middlewares) {
+  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose; // eslint-disable-line no-underscore-dangle
```

Comment:
Okay, so having it separate will allow us to use it in different places then. COol.